### DISCUSSION TITLE
GeoJson Simplestyle

### DISCUSSION THREAD

**Discussion by KorecGroup-AliJafri** [zouyaoji/vue-cesium#619] [Q&A]
Hi! I have seen an implementation of GeoJson simplestyle implementation in this Cesium sandcastle example 
https://sandcastle.cesium.com/?src=GeoJSON%20simplestyle.html

I wanted to ask whether all simplestyle features are supported by adding the required attributes like 'stroke' for lines in a GeoJson when using the VcDatasourceGeojson component? 

An example of the GeoJson: https://github.com/mapbox/simplestyle-spec/blob/master/1.1.0/example.geojson?short_path=d28d809

**Comment by zouyaoji** [comment]
The supported format is fine, but it seems you need to first place the images used in Cesium's directory, for example, /Assets/Textures/maki/rail-metro.png
